### PR TITLE
legacy names of volume plugins have been gone a LONG time

### PR DESCRIPTION
**Pull request by carlory**
#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:

See https://github.com/kubernetes/kubernetes/pull/77537#discussion_r282284291

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Review by r0b2g1t — APPROVED** [review]

**Review by Namanl2001 — APPROVED** [review]
/lgtm
Thanks!

File: pkg/volume/plugins.go [hunk 1]
```diff
@@ -694,13 +694,11 @@ func (pm *VolumePluginMgr) FindPluginBySpec(spec *Spec) (VolumePlugin, error) {
 	return match, nil
 }
 
-// FindPluginByName fetches a plugin by name or by legacy name.  If no plugin
-// is found, returns error.
+// FindPluginByName fetches a plugin by name. If no plugin is found, returns error.
```
[the remark this answers is not in the document]
Comment (carlory) [reply pkg/volume/plugins.go:697]: As @thockin said, legacy names have been gone a LONG time. See https://github.com/kubernetes/kubernetes/pull/77537#discussion_r282284291

**Comment by thockin** [comment]
Thanks!

/lgtm
/approve

**Pull request merged**